swedish/titta-ut/macka/mer-mat/mer-mat.md

```diff
@@ -0,0 +1 @@
+Du tittar i kylskåpet och ser en oöppnad yoghurt med jordgubbssmak. Du äter den i lugn och ro och går och lägger dig igen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.